src/services/FeedProcessor/MultiFeedProcessor.ts

```diff
@@ -24,13 +26,19 @@ export default class MultiFeedProcessor {
       this.cryptoCompareMultiProcessor.apply(feedFetchers),
       this.coingeckoMultiProcessor.apply(feedFetchers),
       this.uniswapV3MultiProcessor.apply(feedFetchers),
+      this.sovrynMultiProcessor.apply(feedFetchers),
     ]);
 
     promisesResults.forEach((result, i) => {
       if (result.status === 'fulfilled') {
         response = mergeArrays(response, result.value);
       } else {
-        const processors = ['cryptoCompareMultiProcessor', 'coingeckoMultiProcessor', 'uniswapV3'];
+        const processors = [
+          'cryptoCompareMultiProcessor',
```

Comment:
let's add `TODO` to put all this in eg enum and use here something like Object.keys(...)